File: LeetCode-Problems/Easy/Target-Array-In-Order.java
//Java
class Solution {
    public int[] createTargetArray(int[] nums, int[] index) {
        ArrayList<Integer> arrList = new ArrayList<Integer>(index.length);
        for (int i = 0; i < index.length; i++){
            arrList.add(index[i], nums[i]);
        }
        int[] returnList = new int[arrList.size()];
        for (int i = 0; i < arrList.size(); i++){
            returnList[i] = arrList.get(i);
        }
        return returnList;
    }
}

File: LeetCode-Problems/Easy/Target-Array-In-Order.py
# Python
class Solution(object):
    def createTargetArray(self, nums, index):
        """
        :type nums: List[int]
        :type index: List[int]
        :rtype: List[int]
        """
        arrList = list()
        
        for x in range(0, len(index)):
            arrList.insert(index[x], nums[x])
        return arrList
